### ISSUE TITLE
Namechange

### ISSUE THREAD

**Issue by rthor** [aranja/tux-autohide#3]
@aranja/tux What about: tux-autohide → tux-scroll-reveal ?

**Comment by eirikurn** [comment]
tux-sticky-edge, tux-sticky-nav?

**Comment by rthor** [comment]
:+1: for tux-sticky, I think that might do it.

**Comment by eirikurn** [comment]
Yeah. I thought about that, but isn't that something different? http://caniuse.com/#feat=css-sticky

**Comment by eirikurn** [comment]
Though maybe, just maybe, the name doesn't need to be perfect...

**Comment by rthor** [comment]
True.... Very true.

**Comment by thorsteinsson** [comment]
I think tux-sticky is way better than tux-autohide, let's change it :+1:
